README.rst

```diff
@@ -72,10 +72,11 @@ then you can run it directly as operation system's command.
 Usage
 -----
 
-RobotLog2DB_ tool requires the RobotFramework ``output.xml`` result file(s) and 
+RobotLog2DB_ requires the Robot Framework result file(s) which contains the 
+output result in XML format (default name is ``output.xml``) and 
 TestResultWebApp's database information for importing.
```

Comment:
Hi Ngoan,
I would expect a link here.
Thank you,
 Thomas

Reply:
Hi Thomas,

I added it in new commit.

Thank you,
Ngoan